features/autoconsent/main.js

```diff
@@ -0,0 +1,82 @@
+// populate test sites
+const testSites = {
```

Comment:
why this format is easier than keeping it in HTML? We use this format elsewhere and it's easier to update in the future?

Reply:
This way is less verbose, and allows test-cases to be easily copied over to/from the tests in the autoconsent project: https://github.com/cliqz-oss/autoconsent/blob/master/addon/test.ts#L82-L91